### DISCUSSION TITLE
HOw you add Device Type Library do as a data souce.

### DISCUSSION THREAD

**Discussion by joaolucasmacedo** [netbox-community/netbox#14077] [Q&A]
Hi everyone.
I've been using devicetype-library as data source on NetBox 3.5 to help import device types.
Now on 3.6 it doesn't work as easy it was.
Has anything changed? Is there anyone using it on NetBox 3.6 to help me?

![image](https://github.com/netbox-community/netbox/assets/66760118/909d498e-d4c6-4a11-a974-e08f635a4dc5)

**Comment by DanSheps** [comment]
What does the job log report?

**Reply by joaolucasmacedo** [reply]
Where can I find the logs?

**Comment by abhi1693** [comment]
On 3.6 you need to install dulwich package yourself now.

**Reply by joaolucasmacedo** [reply]
Thanks! It worked immediately.
